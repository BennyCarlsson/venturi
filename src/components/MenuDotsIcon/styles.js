import styled from "styled-components";
import Colors from "tokens/colors";

export const MenuDotsIconWrapper = styled.div`
  display: grid;
  grid-template-columns: 4px 4px 4px;
  grid-template-rows: 4px 4px 4px;
  column-gap: 5px;
  row-gap: 5px;
`;

export const Dot = styled.div`
  width: 4px;
  height: 4px;
  border-radius: 1px;
  background-color: ${Colors.white};
`;
