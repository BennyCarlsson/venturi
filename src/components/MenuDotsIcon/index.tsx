import React from 'react';
import { Dot, MenuDotsIconWrapper } from './styles';

const MenuDotsIcon = () => {
  return (
    <MenuDotsIconWrapper>
      <Dot />
      <Dot />
      <Dot />
      <Dot />
      <Dot />
      <Dot />
      <Dot />
      <Dot />
      <Dot />
    </MenuDotsIconWrapper>
  );
};

export default MenuDotsIcon;
