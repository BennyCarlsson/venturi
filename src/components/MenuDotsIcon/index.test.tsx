import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import MenuDotsIcon from ".";

describe("<MenuDotsIcon />", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<MenuDotsIcon />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should render without crashing", () => {
    render(<MenuDotsIcon />);
    expect(screen).toBeTruthy();
  });
});
