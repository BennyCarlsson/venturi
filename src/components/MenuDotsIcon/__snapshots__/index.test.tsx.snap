// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuDotsIcon /> should match snapshot 1`] = `
<div
  className="sc-bdfBwQ hvQWz"
>
  <div
    className="sc-gsTCUz hslkzR"
  />
  <div
    className="sc-gsTCUz hslkzR"
  />
  <div
    className="sc-gsTCUz hslkzR"
  />
  <div
    className="sc-gsTCUz hslkzR"
  />
  <div
    className="sc-gsTCUz hslkzR"
  />
  <div
    className="sc-gsTCUz hslkzR"
  />
  <div
    className="sc-gsTCUz hslkzR"
  />
  <div
    className="sc-gsTCUz hslkzR"
  />
  <div
    className="sc-gsTCUz hslkzR"
  />
</div>
`;
